### PR TITLE
MULE-15076: Migrate test to MTF

### PR DESCRIPTION
**Pull request by alegmarra**

### CODE REVIEW

File: src/test/munit/arguments-transformation-test-suite.xml [hunk 1]
```diff
@@ -0,0 +1,344 @@
...
+                </java:args>
+            </java:invoke-static>
+        </munit:execution>
+        <munit:validation>
+            <munit-tools:assert-that doc:name="Assert"
+                                     expression="#[payload]" is="#[MunitTools::nullValue()]"/>
+        </munit:validation>
+    </munit:test>
+
+    <munit:test name="invokeStaticWitNullParameterForPrimitiveValue" expectedErrorType="JAVA:ARGUMENTS_MISMATCH"
```
Comment (diegostrubolini) [inline src/test/munit/arguments-transformation-test-suite.xml]: wit -> with

File: src/test/munit/arguments-transformation-test-suite.xml [hunk 2]
```diff
@@ -0,0 +1,344 @@
...
+                description="MUnit Test" ignore="true">
```
Comment (diegostrubolini) [inline src/test/munit/arguments-transformation-test-suite.xml]: comment, why it is ignored, and relate it to jira

File: src/test/munit/arguments-transformation-test-suite.xml [hunk 3]
```diff
@@ -0,0 +1,344 @@
...
+        </munit:execution>
+        <munit:validation>
+            <munit-tools:assert-that doc:name="Assert"
+                                     expression="#[payload]" is="#[MunitTools::equalTo(28)]"/>
+        </munit:validation>
+    </munit:test>
+
+    <munit:test name="invokeStaticWithNullParameter" description="MUnit Test" ignore="true">
+        <munit:behavior>
+            <set-variable variableName="nullValue"
```
Comment (diegostrubolini) [inline src/test/munit/arguments-transformation-test-suite.xml]: not being used

File: src/test/munit/arguments-transformation-test-suite.xml [hunk 4]
```diff
@@ -0,0 +1,344 @@
...
+                          value="#[java!org::mule::extensions::internal::model::ExecutableElement::getNull()]"/>
+        </munit:behavior>
+        <munit:execution>
+            <java:invoke-static
+                    class="org.mule.extensions.internal.model.AnotherExecutableElement"
+                    method="getSameNumber(int)">
+                <java:args>
+                    #[{ 'number' : vars.nullValue } ]
```
Comment (diegostrubolini) [inline src/test/munit/arguments-transformation-test-suite.xml]: removing this and replacing it with vars.missingValue, should produce the same effect

File: src/test/munit/arguments-transformation-test-suite.xml [hunk 5]
```diff
@@ -0,0 +1,344 @@
...
+                          value="#[java!org::mule::extensions::internal::model::ExecutableElement::getNull()]"/>
+        </munit:behavior>
+        <munit:execution>
+            <java:invoke-static
+                    class="org.mule.extensions.internal.model.AnotherExecutableElement"
+                    method="sayItemFromMapEntry(Map,String,Integer)">
+                <java:args>
+                    #[{
+                    'map': {
+                    '6' : vars.nullValue,
```
Comment (diegostrubolini) [inline src/test/munit/arguments-transformation-test-suite.xml]: same here

File: src/test/munit/arguments-transformation-test-suite.xml (hunk 4 above)
Comment (alegmarra) [reply src/test/munit/arguments-transformation-test-suite.xml]: I removed the var declaration, but leaving the name as `nullValue` seems more descriptive

File: src/test/munit/arguments-transformation-test-suite.xml (hunk 5 above)
Comment (alegmarra) [reply src/test/munit/arguments-transformation-test-suite.xml]: I removed the var declaration, but leaving the name as `nullValue` seems more descriptive

**Pull request merged**